test/tests/functional/pbs_qstat.py

```diff
@@ -55,19 +55,16 @@ def setUp(self):
         a = {'scheduling': 'false', 'flatuid': 'true'}
         self.server1.manager(MGR_CMD_SET, SERVER, a)
         self.server2.manager(MGR_CMD_SET, SERVER, a)
-        self.du.set_pbs_config(self.server.client, confs={'PBS_SERVER': ''})
 
     def test_qstat_req_server(self):
         j = Job(TEST_USER)
-        actualclient = self.server.client
-        self.server.client = self.server1.hostname
         jid = self.server1.submit(j)
-        self.server1.expect(JOB, {'job_state': 'Q'}, id=jid)
         destination = '@%s' % self.server2.hostname
         self.server1.movejob(jobid=jid, destination=destination)
-        self.server.client = actualclient
+        self.du.set_pbs_config(self.server.client, confs={'PBS_SERVER': ''})
```

Comment:
I believe you moved this after movejob() based on @bhroam's comment. Shouldn't we set PBS_SERVER to the previous value in teardown()?

Reply:
Hey @prakashcv13, I believe when ptl starts a new test case it will revert pbs configuration back to out of box. Do you still think we need to take care ?